.buildkite/pipeline.yml

```diff
@@ -84,26 +85,27 @@ steps:
           manual:
             allowed: true
 
-      # Runs inly on the main branch
-      - label: "Unit tests - MacOS 13 ARM"
-        key: "unit-tests-macos-13-arm"
+  - group: "MacOS tests"
+    key: "macos-unit-tests"
+    steps:
+      - label: "Unit tests - MacOS 15 ARM"
         command: ".buildkite/scripts/steps/unit-tests.sh"
-        branches: main
         artifact_paths:
           - "build/TEST-*.html"
           - "build/TEST-*.xml"
           - "build/diagnostics/*"
           - "coverage-*.out"
         agents:
           provider: orka
-          imagePrefix: generic-13-ventura-arm
+          imagePrefix: generic-base-15-arm-002
```

Comment:
What's with the `002`? That comes across as a somewhat arbitrary naming scheme.

Reply:
We have an agreement for Orka images versioning. When we build a new image is built we increment its suffix to make sure we don't affect any pipelines